chart/etcd-backup-restore/templates/etcd-statefulset.yaml

```diff
@@ -71,48 +83,68 @@ spec:
           protocol: TCP
         resources:
 {{ toYaml .Values.resources.etcd | indent 10 }}
-        env:
-        - name: ENABLE_TLS
-          value: "{{ if .Values.backupRestoreTLS }}true{{ else }}false{{ end }}"
-        - name: BACKUP_ENDPOINT
-          value: "http{{ if .Values.backupRestoreTLS }}s{{ end }}://localhost:{{ .Values.servicePorts.backupRestore }}"
-        - name: FAIL_BELOW_REVISION_PARAMETER
-          value: "{{ if .Values.backup.failBelowRevision }}&failbelowrevision={{ int $.Values.backup.failBelowRevision }}{{ end }}"
         volumeMounts:
         - name: {{ .Release.Name }}-etcd
           mountPath: /var/etcd/data/
-        - name: etcd-config-file
-          mountPath: /var/etcd/config/
 {{- if .Values.etcdTLS }}
-        - name: ca-etcd
+        - name: etcd-ca
           mountPath: /var/etcd/ssl/ca
-        - name: etcd-tls
-          mountPath: /var/etcd/ssl/tls
+        - name: etcd-server-tls
+          mountPath: /var/etcd/ssl/server
+        - name: etcd-client-tls
+          mountPath: /var/etcd/ssl/client
 {{- end }}
 {{- if .Values.backupRestoreTLS }}
-        - name: ca-etcdbr
+        - name: backup-restore-ca
           mountPath: /var/etcdbr/ssl/ca
 {{- end }}
       - name: backup-restore
-        command:
-        - etcdbrctl
+        args:
         - server
+{{- if .Values.backup.storageProvider}}
```

Comment:
I like that you have segregated the flags under different headers for better readability. Thanks!